src/SME.SGP.WebClient/src/componentes/TabelaRetratil/style.js

```diff
@@ -68,7 +68,9 @@ export const LinhaTabela = styled.tr`
 		cursor: not-allowed !important;
 
 		&:hover {
```

Comment:
o nome do arquivo deveria ser em português, 'estilo.js'

Reply:
@nogueiraamcom 
é um componente que ja esta criada a um tempo assim como arquivos chamado index e style em todo o sistema que em outros pr passaram, podemos manter assim ou caso for necessário mudar poderia abrir uma task para realizar o ajuste em todo o sistema?

Reply:
blz, deixamos assim pra não precisar mexer em mais lugares, só cuidar dos próximos.